src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -14,6 +17,8 @@ public class ClearCommand extends Command {
 
     @Override
     public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.setAddressBook(new NpcTrack());
```

Comment:
I think that this shouldn't be here, this class only exist to send a warning message. If you add it back in, the user will clear everything without any secondary protection